config/install/tripal.tripalfield_collection.trpcultivate_experiments.yml

```
@@ -1220,3 +1920,211 @@ fields:
+        content_type: biochem_experiment
+        label: Experiment Date
+        type: chado_property_type_default
+        description: The date and time the experiment was performed (e.g. 2024Aug5 3:15pm)
+        cardinality: -1
+        required: false
+        storage_settings:
+            storage_plugin_id: chado_storage
+            storage_plugin_settings:
+                base_table: project
+                prop_table: projectprop
+        settings:
+            termIdSpace: NCIT
+            termAccession: 'C29848'
+        display:
+          view:
+            default:
+              region: content
+              label: above
+              weight: 20
+          form:
+            default:
+              region: content
+              weight: 20
+
+    -   name: bchem_design
+        content_type: biochem_experiment
+        label: Experimental Design
+        type: chado_property_type_default
+        description: Describe the design of your experiment in detail. More specifically, include information on replication, randomization and blocking. Also include the location-year combinations you intend to collect data in. Make sure these match those specifieid in "Experiment Locations" and "Experiment Years".
+        cardinality: -1
+        required: false
+        storage_settings:
+            storage_plugin_id: chado_storage
+            storage_plugin_settings:
+                base_table: project
+                prop_table: projectprop
+        settings:
+            termIdSpace: PROCO
+            termAccession: '0000276'
+        display:
+          view:
+            default:
+              region: content
+              label: above
+              weight: 20
+          form:
+            default:
+              region: content
+              weight: 20
+
+    -   name: bchem_germgenus
+        content_type: biochem_experiment
+        label: Germplasm Genus
+        type: chado_property_type_default
+        description: The genus of the germplasm data was collected for. If data was collected for multiple genus' then each should be it's own entry.
+        cardinality: -1
+        required: false
+        storage_settings:
+            storage_plugin_id: chado_storage
+            storage_plugin_settings:
+                base_table: project
+                prop_table: projectprop
+        settings:
+            termIdSpace: TAXRANK
+            termAccession: '0000005'
+        display:
+          view:
+            default:
+              region: content
+              label: above
+              weight: 20
+          form:
+            default:
+              region: content
+              weight: 20
+
+    -   name: bchem_germspecies
+        content_type: biochem_experiment
+        label: Germplasm Species
+        type: chado_property_type_default
+        description: The species of the germplasm data was collected for. If data was collected for multiple genus' then each should be it's own entry.
```

We may want to update this for all occurrences of this field.
```suggestion
        description: The species of the germplasm that data was collected for (i.e. that the experiment was done on). If data was collected for multiple genus' then each should be it's own entry.
```

---

Just to clarify: "all occurrences of this field" means the following 4 right:

- field_experiment
- growth chamber_experiment
- greenhouse_experiment
- biochem_experiment

---

Yes 👍